src/Components/Teammember.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components";
+
+/**
+ * The Team Member Component
+ * @author [Sam](https://github.com/Samm96)
+ *
+ * React template to create team members.
+ */
+
+// note: remember to replace values with Theme.js props
+
+const Container = styled.div.attrs(() => ({ tabIndex: 0 }))`
+  width: 351px;
```

Comment:
Component for Anya Marshall is wider than the others (affects _teammember__text-container_ as well)